game_tic_tac_toe.py

```diff
@@ -159,11 +184,130 @@ def player_won(self, check_disable):
-            if win_set.issubset(self.check_disable):
-                for elem in win_set:
-                    self.game_btns[elem]['bg'] = '#CCFFFF'
-                return True
-
+            if win_set.issubset(check_disable):
+                if game_state == 'playing':
+                    for elem in win_set:
+                        self.game_btns[elem]['bg'] = '#CCFFFF'
+                    return True
+                elif game_state == 'trying':
+                    return True
         return False
+    
+    def computer_1turn(self):
+        # first turn is always X
+        # chooses move:
+        corners_center= [0,2,6,8,4]
+        move = random.choice(corners_center)
+        # marks the move, disables the button:
+        self.game_btns[move].config(state="disabled", text='X')
+        # counts a turn: 
+        self.count_turns += 1
+        # adds move to set of moves
+        self.disable_numbers_x.add(move)
+        check_disable = self.disable_numbers_x
+        # defines current and next player
+        current_player = self.player_x
+        next_player = self.player_o
+        self.after_move(check_disable, current_player, next_player)
+
+    def computer_turn(self, comp_symbol):
+        '''This function handles the computer turn '''
+        # chooses move:
+        move = self.deciding_computer_move(comp_symbol)
+        
+        # counts computer turn: 
+        self.count_turns += 1      
+        if comp_symbol == "x":
+            # marks the move, disables the button:
+            self.game_btns[move].config(state="disabled", text="X")
+            # adds move to set of moves
+            self.disable_numbers_x.add(move)
+            check_disable = self.disable_numbers_x
+            # defines current and next player
+            current_player = self.player_x
+            next_player = self.player_o
+        else: 
+            self.game_btns[move].config(state="disabled", text="O")
+            # adds move to set of moves
+            self.disable_numbers_o.add(move)
+            check_disable = self.disable_numbers_o
+            # defines current and next player
+            current_player = self.player_o
+            next_player = self.player_x
+        
+        self.after_move(check_disable, current_player, next_player)
+
+    def deciding_computer_move(self, comp_symbol):
+        '''In this function the computer decides the next move
+        Parameters:
+        disable_numbers_o_copy(set): copy of the moves up to this moment of the computer
+        disable_numbers_x_copy(set): copy of the moves up to this moment of the player
+        Return
+        move(int): position on the board'''
+        disable_numbers_o_copy = self.disable_numbers_o.copy()
+        disable_numbers_x_copy = self.disable_numbers_x.copy()
+
+        available_moves = []
+        for i in range(9):
+            if self.game_btns[i]['state'] == 'normal':
+                available_moves.append(i)
```

Comment:
I think you can do `available_moves = set(range(9)) - disable_numbers_o_copy - disable_numbers_x_copy`